docs/content/Getting-Started/Cloud/04-Query-Cube-API.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Query Cube
+permalink: /getting-started/cloud/query-cube
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 5
+isDisableFeedbackBlock: true
+---
+
+## Developer Playground
+
+The Playground is a web-based tool that allows you to query your Cube data
+without writing any code. It's a great way to get started with Cube and explore
+your data.
+
+To open the Playground, open the **Playground** page in Cube Cloud:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Deployment Developer Playground Screen"
+    src="https://ucarecdn.com/43394d9a-a309-46a1-952d-ccbce10078aa/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+Next, you will select the measures and dimensions you want to query, and then
+run the query. Let's do this for the `Orders` cube you generated in the previous
+step.
+
+Click **+ Measure** to display the available measures and add `Orders.count`,
+then click **+ Dimension** for available dimensions and add `Orders.status`.
```

Comment:
I would recommend to do this in two steps: first, select only a dimension, show results; then, add a measure, show the changes. It should be more telling than composing a "complex" query all it once.